### PR TITLE
Add Up

### PR DESCRIPTION
**Pull request by fletcherdodd**

### CODE REVIEW

**Comment by hkamran80** [comment]
Can you provide a screenshot of the SMS 2FA setup and/or login process? I can't find any mention of it on their website.

Please make sure to blur or crop out any personal information.

Screenshot upload guide

File: img/u/up.com.au.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...6c-11 31.2-22 61.6-33 90.5-6.1 16.2-11.8 33-17 48l-.1.6c-6.6 19.4-13.4 39.5-21 58.5a72 72 0 0 1-18.1 28.7 52.4 52.4 0 0 1-42.8 10.2 15 15 0 0 1-11.6-9 41 41 0 0 1-2.8-18.2c.8-12.6 5.2-25.7 10.6-40.3a12421 12421 0 0 1 57.2-152.8c3.9-10 2.6-21.5-3.6-30.4a31 31 0 0 0-27-14.2h-13.6a7014.4 7014.4 0 0 0-36.9 0 50 50 0 0 0-27.1 8.1 381.6 381.6 0 0 0-44.3 38.7c-22 21-49.4 47.2-75.3 64a32.9 32.9 0 0 0-1 54.5 30.5 30.5 0 0 0 17.5 5.5c3.1 0 6-.5 8.5-1a25.6 25.6 0 0 0 2.6-.4c4.9-1 12.8-3 25.7-6.2-6 17.6-11.3 36.2-14.7 55.8a144.3 144.3 0 0 0 32 120.8 140.1 140.1 0 0 0 110 49.2c21.7 0 43.5-4.3 63-12.4a123 123 0 0 0 31.3-19.5 122.7 122.7 0 0 0 59.8 26.7l-1.6 4.6c-12.9 34.7-30.4 82.1-39.8 106a44.2 44.2 0 0 0 4.4 41.8c8.3 12 22 18.9 37.6 18.9h.2l44.7-.4c14 0 26.1-8.5 30.9-21.7l21.9-61 7-19.5 11-30.1 26.7-74.6.2-.6a172 172 0 0 0 107.2 37.3A218.6 218.6 0 0 0 826 682.9c50-46.8 74.4-103.6 72.6-169a153 153 0 0 0-12.8-58.9c-30.2-68.2-89.3-107.3-162.2-107.3Zm-94.6 218.6c1.6-23.7 10-44.8 25-62.8a84 84 0 0 1 61.3-31 45.6 45.6 0 0 1 3.4 0c18.1 0 32.8 9.6 41.3 27.1 5 10.2 7 19.6 5.9 27.8-4.7 34.7-20 59.8-47 76.8a78.6 78.6 0 0 1-42.4 12.8h-.4a45.3 45.3 0 0 1-44-32.6 76.6 76.6 0 0 1-3-16.4v-1.1l-.1-.5Z"/></svg>
```
Comment (hkamran80) [inline img/u/up.com.au.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 1000 1000"><path fill="#FF7A64" d="M530.3 98.8 919.7 771a19.7 19.7 0 0 1-17 29.5H123.9a19.7 19.7 0 0 1-17-29.5L496.3 98.8c7.5-13 26.4-13 34 0Z"/><path fill="#fff" d="M878 514.5c-.5-18.7-4.1-35.4-11-51.2-30.9-69.7-95-104.5-171.8-93a209 209 0 0 0-38 9.4c-4.3-5.5-10.4-7.9-19.7-7.9h-46c-5 0-9.8 3.3-11.7 8.5a33740 33740 0 0 1-83.4 227.4c-3.3 8.7-16.5 36.7-48.1 31.3-7-1.1-13.2-5.4-17.2-11.6a23.6 23.6 0 0 1-3-19.1l.8-2.7c1.9-6.5 4-14 6.5-20.6l73.2-193a12.4 12.4 0 0 0-1.4-11.4 12.5 12.5 0 0 0-10.2-5.3c-23.3-.3-47.6-.2-54.1 0a23.9 23.9 0 0 0-24.2 17 4123 4123 0 0 1-33.2 90.8c-6 16.1-11.8 33-16.9 48-6.7 19.7-13.6 40.1-21.5 59.6a89 89 0 0 1-23.9 36.7 73.5 73.5 0 0 1-60 14.5c-27.7-5.7-31.5-33-30.6-48.3a165 165 0 0 1 11.7-46.2c12.6-34.5 38.1-102.7 57.4-153.1 1.5-3.8 1-8-1.3-11.4-2.4-3.4-4.7-5.4-10.2-5.4-6 0-44.4-.2-49.8 0a30 30 0 0 0-16.6 4.7c-10.8 7.2-25 20.8-41.5 36.5-22.6 21.6-50.8 48.5-78.2 66.3a12.5 12.5 0 0 0-5.7 10.1c0 4.1 2 8 5.3 10.4 3 2.1 5.9 2 11 1.1l1.2-.2c7.7-1.6 24.8-6 63-15.7a456.8 456.8 0 0 0-27.4 88.7 124.5 124.5 0 0 0 27.6 104.2c39.8 47.6 105.3 49.5 149.4 31a114.8 114.8 0 0 0 39.7-29.3c20.8 27.5 57.8 39.1 88.3 36l-8.2 22.4-3.3 8.7a8192.7 8192.7 0 0 1-39.8 106.4 24 24 0 0 0 2.1 22.8c4.4 6.3 11.9 10 20.8 10l44.8-.3c5.3 0 9.9-3.3 11.7-8.2 7.3-20.3 18-50.3 29-80.5l10.9-30.2a48702 48702 0 0 0 26.8-74.5L562 668a139.1 139.1 0 0 0 15.4 15.4 153.6 153.6 0 0 0 129 35.3A198.7 198.7 0 0 0 811.8 668c45.6-42.5 67.9-94.2 66.2-153.5Zm-92 15.7c-5.4 40.4-24.3 71.1-56.3 91.3a98.7 98.7 0 0 1-53.3 16h-.4a66 66 0 0 1-63.4-46.9 98.5 98.5 0 0 1-4-22l-.2-1.8a127.3 127.3 0 0 1 30-76.4 104 104 0 0 1 75.5-38.2 65.3 65.3 0 0 1 4.8-.2 65 65 0 0 1 59.6 38.7 69.3 69.3 0 0 1 7.8 39.5Z"/><path fill="#242430" d="M723.5 368.1c64 0 116.5 34.3 143.4 95.2 7 15.8 10.6 32.5 11.1 51.2 1.7 59.3-20.6 111-66.2 153.5A199 199 0 0 1 678 721c-37 0-70.7-12.6-100.5-37.7a134.3 134.3 0 0 1-15.4-15.4l-10.7 30-26.8 74.4-10.9 30.2a89312.3 89312.3 0 0 0-29 80.5 12 12 0 0 1-11.6 8.2l-44.9.3c-8.9 0-16.3-3.6-20.7-10a24 24 0 0 1-2.1-22.8c9.2-23.7 26.2-69.5 39.8-106.3l3.2-8.8 8.3-22.3a95.1 95.1 0 0 1-9.7.5c-28.3 0-60-12-78.6-36.6a116.1 116.1 0 0 1-39.8 29.4 145.2 145.2 0 0 1-55 10.9c-33.8 0-69.2-11.9-94.4-42a124.5 124.5 0 0 1-27.5-104.1 452.9 452.9 0 0 1 27.4-88.7c-38.2 9.7-55.4 14-63 15.6a6.3 6.3 0 0 1-1.3.3c-2 .4-3.8.6-5.3.6a9 9 0 0 1-5.6-1.8 12.3 12.3 0 0 1-5.3-10.3 12.4 12.4 0 0 1 5.6-10.2c27.5-17.8 55.6-44.7 78.3-66.3 16.5-15.7 30.7-29.3 41.5-36.5a30 30 0 0 1 16.5-4.7h49.9c5.4 0 7.8 2 10 5.4a12.3 12.3 0 0 1 1.4 11.4C282.4 444.6 257 513 244.4 547.3a164 164 0 0 0-11.8 46.2c-1 15.3 3 42.6 30.7 48.4a70.1 70.1 0 0 0 14.1 1.4 73 73 0 0 0 45.9-16 90 90 0 0 0 23.9-36.7 1092 1092 0 0 0 21.4-59.6c5.1-14.9 10.8-31.9 17-48a3963 3963 0 0 0 33.1-90.8 24.1 24.1 0 0 1 24.2-17 1968.5 1968.5 0 0 1 54.1 0 12.5 12.5 0 0 1 11.6 16.7L435.4 585c-2.5 6.6-4.6 14-6.5 20.5l-.8 2.8a23.6 23.6 0 0 0 3 19c4 6.3 10.2 10.5 17.2 11.7 2.7.5 5.4.7 7.8.7 26 0 37.3-24 40.3-32a33740 33740 0 0 0 83.4-227.4c2-5.2 6.6-8.4 11.8-8.5h46c9.2 0 15.3 2.4 19.6 8a206.2 206.2 0 0 1 38-9.5c9.7-1.5 19-2.2 28.3-2.2ZM676 637.4h.4c18.7 0 36.6-5.3 53.3-16 32-20.1 51-50.8 56.4-91.2a69 69 0 0 0-7.8-39.5 65 65 0 0 0-59.6-38.7 65.3 65.3 0 0 0-4.8.2 104 104 0 0 0-75.6 38.2 127.3 127.3 0 0 0-29.9 76.4l.2 1.8.5 6.4c.9 5.8 2 11 3.5 15.6a66 66 0 0 0 63.4 46.8Zm47.5-289.7a213 213 0 0 0-31.3 2.4 228.9 228.9 0 0 0-30.8 6.8 48.8 48.8 0 0 0-23.9-5.6h-46c-14 .1-26 8.7-31 22C560 375 497 548 477.3 600.4c-2.1 5.6-8.4 18.7-21.1 18.7a26 26 0 0 1-4.4-.4 5.2 5.2 0 0 1-3.3-2.4c-.7-1.1-.8-2-.7-2.5l.8-2.8c1.8-6.1 3.8-13 6-18.9l73.2-193A33 33 0 0 0 524 369a33 33 0 0 0-26.7-14.2l-29.8-.2-24.8.1a44.7 44.7 0 0 0-43.3 30.6c-11 31.2-22 61.6-33 90.5-6.1 16.2-11.8 33-17 48l-.1.6c-6.6 19.4-13.4 39.5-21 58.5a72 72 0 0 1-18.1 28.7 52.4 52.4 0 0 1-42.8 10.2 15 15 0 0 1-11.6-9 41 41 0 0 1-2.8-18.2c.8-12.6 5.2-25.7 10.6-40.3a12421 12421 0 0 1 57.2-152.8c3.9-10 2.6-21.5-3.6-30.4a31 31 0 0 0-27-14.2h-13.6a7014.4 7014.4 0 0 0-36.9 0 50 50 0 0 0-27.1 8.1 381.6 381.6 0 0 0-44.3 38.7c-22 21-49.4 47.2-75.3 64a32.9 32.9 0 0 0-1 54.5 30.5 30.5 0 0 0 17.5 5.5c3.1 0 6-.5 8.5-1a25.6 25.6 0 0 0 2.6-.4c4.9-1 12.8-3 25.7-6.2-6 17.6-11.3 36.2-14.7 55.8a144.3 144.3 0 0 0 32 120.8 140.1 140.1 0 0 0 110 49.2c21.7 0 43.5-4.3 63-12.4a123 123 0 0 0 31.3-19.5 122.7 122.7 0 0 0 59.8 26.7l-1.6 4.6c-12.9 34.7-30.4 82.1-39.8 106a44.2 44.2 0 0 0 4.4 41.8c8.3 12 22 18.9 37.6 18.9h.2l44.7-.4c14 0 26.1-8.5 30.9-21.7l21.9-61 7-19.5 11-30.1 26.7-74.6.2-.6a172 172 0 0 0 107.2 37.3A218.6 218.6 0 0 0 826 682.9c50-46.8 74.4-103.6 72.6-169a153 153 0 0 0-12.8-58.9c-30.2-68.2-89.3-107.3-162.2-107.3Zm-94.6 218.6c1.6-23.7 10-44.8 25-62.8a84 84 0 0 1 61.3-31 45.6 45.6 0 0 1 3.4 0c18.1 0 32.8 9.6 41.3 27.1 5 10.2 7 19.6 5.9 27.8-4.7 34.7-20 59.8-47 76.8a78.6 78.6 0 0 1-42.4 12.8h-.4a45.3 45.3 0 0 1-44-32.6 76.6 76.6 0 0 1-3-16.4v-1.1l-.1-.5Z"/></svg>
```

File: entries/u/up.com.au.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+  "Up": {
+    "domain": "up.com.au",
+    "tfa": [
+      "sms"
```
Comment (hkamran80) [inline entries/u/up.com.au.json:5]: Can you provide a screenshot of the SMS 2FA setup and/or login process? I can't find any mention of it on their website.

Please make sure to blur or crop out any personal information.

Screenshot upload guide

**Review by hkamran80 — CHANGES_REQUESTED** [review]

File: entries/u/up.com.au.json (hunk 2 above)
Comment (fletcherdodd) [reply entries/u/up.com.au.json:5]: image

**Review by hkamran80 — APPROVED** [review]

**Pull request merged**